README.md

```diff
@@ -215,10 +223,6 @@ if (result.error !== null) {
 ## Roadmap Features
 * start/end string locations for URL attribute values ([parse5#43](https://github.com/inikulin/parse5/issues/43))
 * option to exclude keywords from URLs (facebook.com, etc)
-* response cache to avoid checking same URL twice (even with different hashes)?
-  * per class instance?
-  * could copy response from first checked link for each successive link's completeness
-  * provide method to clear cache
 * change order of checking to: tcp error, 4xx code (broken), 5xx code (undetermined), 200
```

Comment:
Done. I'll probably be stepping back from this a bit since I'll be moving projects, but would love to do some more PRs on this if I get a chance :)